teams/teams-ps/teams/Get-CsTeamsVirtualAppointmentsPolicy.md

```diff
@@ -0,0 +1,121 @@
+---
+external help file: Microsoft.Teams.Policy.Administration.Cmdlets.Core.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://learn.microsoft.com/powershell/module/skype/get-csteamsvirtualappointmentspolicy
+schema: 2.0.0
+ms.author: erocha
```

Comment:
please also add "author" metadata, that is your GitHub username

Reply:
addressed.